src/Compilers/CSharp/Test/Symbol/SymbolDisplay/SymbolDisplayTests.cs

```diff
@@ -3997,6 +3997,112 @@ void P(E e = (E)(-2), E f = (E)(-1), E g = (E)0, E h = (E)(-3))
                 SymbolDisplayPartKind.Punctuation); //)
         }
 
+        [Fact]
+        public void DefaultParameterValues_NullableEnum()
```

Comment:
>DefaultParameterValues_NullableEnum [](http://example.com/codeflow?start=20&length=35)

Consider adding a corresponding test for VB.

Reply:
vb doesn't seem to have existing tests on enum parameter values so I am also adding an enum and a flags enum test.